src/controls/richText/RichText.module.scss

```diff
@@ -97,6 +97,19 @@
   color: "[theme:neutralLighterAlt, default:#{$ms-color-neutralLighterAlt}]";
 }
 
+.toolbarSubmenuCaretLT {
```

Comment:
Why do you need this non-standard caret style for the dropdown?
Why don't just use a default one?